docsrc/admin/installation.md

```diff
@@ -50,34 +76,50 @@ If you feel SELinux is important to the security of your server, please start a
+* Modify `playbook.yaml` and `../secrets.yaml` to customize them to the specifics of your site. The files that you
+  copied contain example configuration values that should be replaced with real email addresses, usernames, *etc.* The
+  files include comments that explain the purpose of each configuration option. If you are not ready to set up your real
+  Data Library server but merely want to practice the installation process, *e.g.* in a virtual machine, you can use the
+  example files without modification.
 
-* Commit your customizations and push them to your git server for safe keeping; back up `secrets.yaml` by other means, such as copying it to another machine.
+* Commit your customizations and push them to your git server for safe keeping; back up `secrets.yaml` by other means,
+  such as copying it to another machine.
 
-Never edit the contents of the `ansible_collections` directory. All customization should be made in the configuration files that you copied from the template. In the future when it comes time to upgrade to a newer version of the DL software, you will run the `ansible-galaxy` command again and commit the new version to your configuration repository. Don't upgrade without checking the release notes first, because in some cases an upgrade may require manual migration steps. (At this writing, there are no upgrade release notes because this is the playbook's initial release.)
+Never edit the contents of the `ansible_collections` directory. All customization should be made in the configuration
+files that you copied from the template. In the future when it comes time to upgrade to a newer version of the DL
+software, you will run the `ansible-galaxy` command again and commit the new version to your configuration repository.
+Don't upgrade without checking the release notes first, because in some cases an upgrade may require manual migration
+steps. (At this writing, there are no upgrade release notes because this is the playbook's initial release.)
 
 ## Run the ansible playbook
-Now we are ready to run the playbook, which will download, configure, and install the Data Library software using the parameters you defined in the configuration files.
+
+Now we are ready to run the playbook, which will download, configure, and install the Data Library software using the
+parameters you defined in the configuration files.
 
 From the root directory of the configuration repository, run the following command:
 
-    ansible-playbook \
-        --ask-become-pass \
-        -i inventory.cfg \
-        -e @../secrets.yaml \
-        -e run_update_script=yes \
-        playbook.yaml
+    ansible-playbook -v --ask-become-pass -i inventory.cfg -e @../secrets.yaml -e run_update_script=yes playbook.yaml
```

Comment:
Why this change? I find the previous version more readable.

Reply:
You're right.  I did it once when I was copy/pasting the command to edit and forgot to change it back.

Reply:
Still waiting for this to be reverted.